charts/values.yaml

```diff
@@ -65,7 +65,7 @@ nodeSelector: {}
 # The "" storageClass will use the default storage class for your cluster. (gp2 for EKS, standard for Minikube)
 # If set the enabled to false, it will use EmptyDir volume
 persistence:
-  enabled: true
+  enabled: false
```

Comment:
Should this be false?

Reply:
Actually if it is by default true and you do not specify the StorageClass (currently empty) it may lead to an issue if there is no StorageClass marked as default.
And it is a standard practice to set the persistency false by default.

Reply:
I'll revert this change as it was not part of the related story.

Reply:
Done I've set back the persistence to true by default